app/controllers/posts_controller.rb

```diff
@@ -205,7 +205,7 @@ def update
   end
 
   def close
-    unless check_your_privilege('flag_close', nil, false)
+    unless check_your_privilege('flag_close', nil, false) || @post.user.id === current_user.id
```

Comment:
You're in Ruby, not JS - only two = needed here :)

Reply:
Force of habit 😂. Will update tomorrow